spec/ddtrace/gem_packaging_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rubygems'
+require 'rubygems/package'
+require 'rubygems/package/tar_reader'
+
+RSpec.describe 'gem release process (after packaging)' do
+  # TODO: This will need to be updated for the 2.0 branch
```

Comment:
Given the release task copies all gems under `/pkg` maybe we could run this check against all `*.gem` files in pkg atm? May be slightly annoying in that it may match on things not produced by the build but those things would be invalid gems and should be deleted sooner rather than later anyway?

Reply:
Good point. I still kept a check that the file we expect is there (just in case we run the spec out-of-order or something like that), but I've expanded it to check every file: https://github.com/DataDog/dd-trace-rb/pull/3531/commits/b059258dbc5652ae5a359c2a47bcc92c87c9b2a7

Reply:
Yeah keeping the file check is smart indeed!